packages/ERTP/src/paymentLedger.js

```diff
@@ -84,9 +87,37 @@ export const vivifyPaymentLedger = (
   elementSchema,
   optShutdownWithFailure = undefined,
 ) => {
-  const getBrand = () => brand;
+  /** @type {Brand} */
+  const brand = vivifyFarInstance(issuerBaggage, `${name} brand`, BrandI, {
+    isMyIssuer(allegedIssuer) {
+      return allegedIssuer === issuer;
```

Comment:
The old version used `E.when()` here. What's the reason to stop supporting Promises for issuers?  (and later for payments.)

Reply:
I'm not! This is part of what's so cool about interface guards. The guard for this method says
```js
  isMyIssuer: M.callWhen(M.await(IssuerShape)).returns(M.boolean()),
```

The `M.callWhen(` means that the call to the unprotected method can be delayed and arrive out of order (as opposed to `M.call(` which must be delivered to the unprotected method synchronously in order). The `M.await(pattern)` means to delay invoking the unprotected method until the possible promise in that argument position fulfills. Once it does, then check its fulfillment against the argument pattern. Only once all that passes, then invoke the unprotected method with an argument in that position that satisfies that pattern.

In this case, the pattern `IssuerShape` merely enforces that the argument is a remotable. But this means that the unprotected method can not be written assuming a remotable in that position!

Reply:
(Note: I just fixed a typo that inverted meaning: "not" -> "now")

Reply:
That seems like significant info about the semantics of the method. What do you think about moving those guards next to the methods?

Reply:
While I agree, I don't see a way to do that while still having the method guards be part of their interface guard. Instead I added some commentary at the top of the method bodies where it seemed most relevant. Resolving, because I see nothing better to do. But please unresolve if you have a suggestion.